docs/advanced-selectors.md

```diff
@@ -0,0 +1,1357 @@
+# Advanced Selectors
```

Comment:
See the [rendered page](https://github.com/kubeshop/tracetest/blob/docs/selectors/docs/advanced-selectors.md) instead of the source code. You will thank me later.